### DISCUSSION TITLE
Saving state in svelte flow

### DISCUSSION THREAD

**Discussion by bluecorridor** [xyflow/xyflow#3848] [Q&A]
In svelte flow, how would one save the current content of a flow? Would we just make a copy of the nodes and edges property of the SvelteFlow component? (I didn’t see a toObject function)

**Comment by bcakmakoglu** [comment]
I think `useSvelteFlow` has a `toObject` function although it seems undocumented. 
https://github.com/xyflow/xyflow/blob/main/packages/svelte/src/lib/hooks/useSvelteFlow.ts#L439
